src/openzaak/components/zaken/api/viewsets.py

```diff
@@ -2110,9 +2110,33 @@ def get_object(self, uuid):
 
         return obj
 
-    def _pre_post(self, request, instance):
+    def _pre_post(self, instance):
+        self._check_zaak_closed(instance, "zaken")
+
+        zaak_data = ZaakSerializer(instance, context={"request": self.request}).data
+
+        if not self.request.jwt_auth.has_auth(
+            scopes=SCOPE_STATUSSEN_TOEVOEGEN | SCOPEN_ZAKEN_HEROPENEN,
+            zaaktype=zaak_data["zaaktype"],
+            vertrouwelijkheidaanduiding=zaak_data["vertrouwelijkheidaanduiding"],
+            init_component="zaken",
+        ):
+            if instance.status_set.exists():
+                msg = f"Met de '{SCOPE_ZAKEN_CREATE}' scope mag je slechts 1 status zetten"
+                raise PermissionDenied(detail=msg)
+
+        if not self.request.jwt_auth.has_auth(
+            scopes=SCOPEN_ZAKEN_HEROPENEN,
+            zaaktype=zaak_data["zaaktype"],
+            vertrouwelijkheidaanduiding=zaak_data["vertrouwelijkheidaanduiding"],
+            init_component="zaken",
+        ):
```

Comment:
Isn't this the same as what `self._check_zaak_closed(instance, "zaken")` should do?

Reply:
check, thanks for explaining :+1: